### PR TITLE
Nrf52 support

### PR DESCRIPTION
**Pull request by TMRh20**
Support NRF52 devices

### CODE REVIEW

File: RF24Ethernet.h [hunk 1]
```diff
@@ -206,6 +229,8 @@ class RF24EthernetClass
 
 extern RF24EthernetClass RF24Ethernet;
 
+typedef RF24EthernetClass RF52EthernetClass;
+
 #endif // RF24Ethernet_h
```
Comment (2bndy5) [inline RF24Ethernet.h:234]: The extra indent of pre-processor cmds is due to the fact that clang-format expects this `#endif` to be the absolute last statement in the header file.

Moving the `#endif` to EoF shouldn't have any consequences.

**Review by 2bndy5 — COMMENTED** [review]
Should probably add a notice in the README that these changes prohibit nRF24L usage on boards based on the nRF5x family.

File: RF24Ethernet.h (hunk 1 above)
Comment (TMRh20) [reply RF24Ethernet.h]: Updated and note added to readme

File: README.md [hunk 2]
```diff
@@ -3,18 +3,21 @@
...
-### Current stage: Functional & Stable (Updated 2021)
+### Updated 2023:
 
 RF24Ethernet requires the RF24, RF24Network and RF24Mesh libraries
 
 Documentation & Setup Info: http://nRF24.github.io/RF24Ethernet
 
 Downloads: http://tmrh20.github.io/
 
+Note: Recent changes to support NRF5x boards prevent usage of RF24 devices with NRF5x boards. Users must use the internal radio with RF24Ethernet.
```
Comment (2bndy5) [inline README.md:16]:
```suggestion
Note: Recent changes to support NRF5x boards prevent usage of RF24 devices with NRF5x boards. Users must use the internal radio with RF24Ethernet.
See the [nrf_to_nrf Arduino library](https://github.com/TMRh20/nrf_to_nrf).
```

**Review by 2bndy5 — APPROVED** [review]
I'm assuming you tested these changes with HW. This TCP/IP stuff is beyond my skill set.

**Comment by TMRh20** [comment]
Yup, tested on a Feather and XIAO Sense.

**Pull request merged**